src/components/views/elements/AppTile.tsx

```diff
@@ -241,7 +241,11 @@ export default class AppTile extends React.Component<IProps, IState> {
     private iframeRefChange = (ref: HTMLIFrameElement): void => {
         this.iframe = ref;
         if (ref) {
-            if (this.sgWidget) this.sgWidget.start(ref);
+            try {
+                if (this.sgWidget) {
+                    this.sgWidget.start(ref);
+                }
+            } catch (e) { logger.log("Failed to start widget", e); }
```

Comment:
When we use blocks, we generally prefer to use the lines it gives us. Also, error for errors :)
```suggestion
            } catch (e) { 
                logger.error("Failed to start widget", e);
            }
```